.github/scripts/collect-test-robot.sh

```diff
@@ -62,6 +62,9 @@ echo '/tmp/core.%p' > /proc/sys/kernel/core_pattern
 #remove git dubious ownership
 /usr/bin/git config --global --add safe.directory $PWD
 
+echo "###### git clone opentelemetry-proto  #######"
+git clone https://github.com/open-telemetry/opentelemetry-proto.git opentelemetry-proto
```

Comment:
you should use shallow clone to download only what you want and faster : `--depth=1 --single-branch --branch=<branch>`